website/content/en/preview/upgrade-guide.md

```diff
@@ -11,6 +11,26 @@ Use your existing upgrade mechanisms to upgrade your core add-ons in Kubernetes
 
 To make upgrading easier we aim to minimize introduction of breaking changes with the followings:
 
+## Compatibility Matrix  
+| Kubernetes Version | Minimum Karpenter Version |
+| :---: | :---: |
+|   1.20    |   v0.2.6+   |
```

Comment:
Should we go back this far? If we go back this far, we would have to have a max version semantic as well since the we certainly don't support `1.20` on the latest version of Karpenter due to API compatability

Reply:
In that case, we can document EKS supported version